src/Components/Movies.js

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import axios from "axios";
+
+class Movies extends React.Component {
+  state = { movies: [], movieValue: "", movieObj: {} };
+
+  fecthMovie = async () => {
```

Comment:
Because this is fetching all movies, we want to make names like this plural. fetchMovies